src/Features/Core/Portable/ConvertIfToSwitch/AbstractConvertIfToSwitchCodeRefactoringProvider.cs

```diff
@@ -45,7 +45,7 @@ public sealed override async Task ComputeRefactoringsAsync(CodeRefactoringContex
 
             var operations = parentBlock.Operations;
             var index = operations.IndexOf(ifOperation);
-            var analyzer = CreateAnalyzer(document.GetLanguageService<ISyntaxFactsService>(), ifStatement.SyntaxTree.Options);
+            var analyzer = CreateAnalyzer(document.GetLanguageService<ISyntaxFactsService>()!, ifStatement.SyntaxTree.Options);
```

Comment:
You can do document.Project.LanguageServices.GetRequiredService() which returns non-nullable (and throws if ever this is somehow broken.)

Reply:
@alrz you probably will need to update from the latest master.  I fixed this last night when doing infra work, CI was broken
https://github.com/dotnet/roslyn/blob/master/src/Features/Core/Portable/ConvertIfToSwitch/AbstractConvertIfToSwitchCodeRefactoringProvider.cs#L48

If you want to change it to getrequired as Jason suggested I wouldn't mind :)